app/src/util/form-schema/manual-input-hierarchy.json

```diff
@@ -3660,10 +3667,13 @@
         {
           "id": "aviation-vehicle-type",
           "options": [
+            "vehicle-type-all",
```

Comment:
Does `all` need to be marked as an exclusive field here? E.g. like it is for `vehicle-type-all` here: https://github.com/Open-Earth-Foundation/CityCatalyst/pull/1313/files#diff-895041166ff96cb9be19e406047e22e8fea828cb42494b1ce2be094a83fe6433R3616